docs/overview/first-steps.md

```diff
@@ -44,10 +44,12 @@ Or adding template name and package manager as arguments:
 expressots new <project-name> -t <template-name> -p <package-manager>
 ```
 
-Templates available:
+:::tip
+Expresso TS offers two options for project templates:
 
-- Non-opinionated: Want to have the liberty to build and structure your project as you wish? Non opinionated template offers a very small footprint, with only the necessary files to get you started.
-- Opinionated: Opinionated template is made for more complex projects, as it provides a more layered architecture.
+- **Non-opinionated**: Want to have the liberty to build and structure your project as you wish? Non opinionated template offers a very small footprint, with only the necessary files to get you started.
```

Comment:
You can remove the bold from Non-Opinionated and Opinionated as well. The reason is that based on the [Doc How To Read](https://expresso-ts.com/docs/hello), bold text are used to highlight links